Decred Wallet/Utils/LocalizedStrings.swift

```diff
@@ -229,8 +229,6 @@ struct LocalizedStrings {
     static let unexpectedError = NSLocalizedString("unexpectedError", comment: "")
     static let sendingTransaction = NSLocalizedString("sendingTransaction", comment: "")
     static let failedTransaction = NSLocalizedString("failedTransaction", comment: "")
-    static let walletAddressShort = NSLocalizedString("walletAddressShort", comment: "")
-    static let walletAddressLong = NSLocalizedString("walletAddressLong", comment: "")
```

Comment:
Should probably delete these from the respective string localization files as well.

Reply:
resolved in `b2c1b1f`